sensor/tests/connection/k8sreconciliation/reconcile_test.go

```diff
@@ -18,6 +19,9 @@ var (
 )
 
 func Test_SensorReconcilesKubernetesEvents(t *testing.T) {
+	if buildinfo.ReleaseBuild {
+		t.Skip("ROX_PREVENT_SENSOR_RESTART_ON_DISCONNECT is not enabled on release builds")
+	}
 	t.Setenv("ROX_PREVENT_SENSOR_RESTART_ON_DISCONNECT", "true")
```

Comment:
```suggestion
        s.T().Setenv(features.PreventSensorRestartOnDisconnect.EnvVar(), "true")
        if !features.PreventSensorRestartOnDisconnect.Enabled() {
          s.T().Skip("Skip tests when ROX_PREVENT_SENSOR_RESTART_ON_DISCONNECT is disabled")
          s.T().SkipNow()
        }
```

This should skip for release but also in any case where it's disabled (or can't be enabled) for some reason

Reply:
Sounds good. Thanks!